src/plugins/drawing-tool/components/drawing-tool.tsx

```diff
@@ -38,6 +38,8 @@ const DrawingToolComponent: React.FC<IProps> = (props) => {
     });
     hotKeys.current.register({
       "cmd-v": handlePaste, //allows user to paste image with cmd+v
+      "delete": handleDelete, // I'm not sure if this will handle "Del" IE 9 and Edge
```

Comment:
this comment makes me realize I don't know what browsers we support! Do we support IE 9 and Edge?

Reply:
We used to, but we don't anymore unless a project explicitly needs it.   

In this case I left the comment because it came from the previous code which was using the name of the key `"Del"`.  The hotKeys helper uses key codes instead of names, so it isn't clear if it would work. I did just test on a chrome book with a full number pad and both delete and backspace work.